src/en/en.json

```
@@ -61,6 +61,8 @@
     "demo": "/en/demo",
     "releaseNotes": "https://github.com/cds-snc/gcds-components/blob/main/CHANGELOG.md",
     "privacy": "/en/privacy-notice",
+    "termsofuse": "/en/terms-of-use",
```

Since we use camelCase on the other keys here could we do that for this one too?
```suggestion
    "termsOfUse": "/en/terms-of-use",
```

SLA below doesn't seem like it'll need it though